### PR TITLE
Surfacing first ceremony (Anand Sahib with Salok)

### LINKED ISSUE #514: Backend work for ceremonies.
Separating the backend work fro #428 that's due release in 5.0

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Anand sahib with salok works now but there are some problems.

* I don't have transliterations in realm DB I was given.
* There are still no ceremonies in sqlite db i have.
* The translations and teeka were also changed in schema, so I updated existing code. I just hope that was intentional (because it will break things with existing db).-

*update:*
fixes #514 #515 

Database structure in realm and sqlite were different so had to add conditionals to conform. 🙏

### CODE REVIEW

**Comment by navdeepsinghkhalsa** [comment]
- Transliterations will now be done on-the-fly using `anvaad-js` https://www.npmjs.com/package/anvaad-js
- Please make sure you have the most recent DB. @ManjotS added them last week
- Yes. It was intentional :)

**Comment by navdeepsinghkhalsa** [comment]
GitHub's introduced a cool new feature called Draft Pull Requests. Could you please use those in the future instead of `[WIP]`? :)
https://github.blog/2019-02-14-introducing-draft-pull-requests/

File: www/js/shortcut_tray.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "anandSahib": {
         "label": "Anand Sahib",
```
Comment (tsingh777) [inline www/js/shortcut_tray.json]: just wonder, should we some how specify this is 6 pauri anand sahib (with salok)?

Comment (tsingh777) [reply www/js/shortcut_tray.json]: cc. @maneetpaul

File: www/js/banidb/realm-search.js [hunk 2]
```diff
@@ -161,6 +161,30 @@ const loadShabad = ShabadID => (
...
+ * @returns {object} Returns array of objects for each line
+ * @example
+ *
+ * loadCeremony(3);
+ * // => [{ Gurmukhi: 'jo gurisK guru syvdy sy puMn prwxI ]', ID: 31057 },...]
+ */
+
+const loadCeremony = ceremonyID => (
+  new Promise((resolve, reject) => {
+    Realm.open(realmConfig)
```
Comment (navdeepsinghkhalsa) [inline www/js/banidb/realm-search.js]: Could you please do the check to see if it's been initialized first?

Comment (inderpreetsingh) [reply www/js/banidb/realm-search.js]: Sure. Sorry, missed it 🙈

File: www/js/shortcut_tray.json (hunk 1 above)
Comment (maneetpaul) [reply www/js/shortcut_tray.json]: @inderpreetsingh can we label it "Anand Sahib + Salok"

File: www/js/controller.js [hunk 3]
```diff
@@ -477,7 +478,20 @@ module.exports = {
     global.platform.ipc.send('clear-apv');
   },
 
-  sendLine(shabadID, lineID, Line, rows) {
+  remapLine(rawLine) {
+    const Line = Object.assign(rawLine, {});
```
Comment (navdeepsinghkhalsa) [inline www/js/controller.js]: `Object.assign({}, rawLine);`

Comment (inderpreetsingh) [reply www/js/controller.js]: I am not sure why but that didn't work for me. 🙏 🤔 

Looking into it again.

**Pull request merged**